### PR TITLE
Type explorer

### PR DESCRIPTION
**Pull request by freiksenet**

### CODE REVIEW

File: python_modules/dagit/dagit/webapp/src/Pipeline.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import SpacedCard from "./SpacedCard";
 import Config from "./Config";
 import Solid from "./Solid";
 import PipelineGraph from "./graph/PipelineGraph";
+import ConfigEditor from "./configeditor/ConfigEditor";
```
Comment (schrockn) [inline python_modules/dagit/dagit/webapp/src/Pipeline.tsx]: Hmmm perhaps let's rename all of this to TypeExplorer

**Comment by schrockn** [comment]
Btw I just pushed https://github.com/superconductive/clarify-data-warehouse-201803/tree/version-0.2.4 to the superconductive repo. So now this explorer can be tested on those pipelines.

**Comment by bengotow** [comment]
This looks great! I'm going to rebase this on top of the new UI - it looks like it'll be pretty straightforward, but I renamed some of the classes and changed the App component a bit.

**Comment by bengotow** [comment]
Hey folks - I've rebased this on master and re-incorporated the type explorer as a new tab of the right sidebar and styled it to match the new stuff. Curious to see how this feels - I'm hesitant to add another sidebar on the other side, and I was hoping we could go for an Xcode-style tabbed panel where you can choose which sort of additional info it provides. (See screenshots for reference)

image

image

Idea:
image

image

**Comment by bengotow** [comment]
I've gone ahead and done a manual merge to keep the history intact since @freiksenet broke things out in to some nice commits. 👍

**Pull request closed without merging**